src/facebook.py

```diff
@@ -87,8 +87,11 @@ class GraphAPI(object):
     for the active user from the cookie saved by the SDK.
     """
 
-    def __init__(self, access_token=None):
+    def __init__(self, access_token=None, version=None):
         self.access_token = access_token
+        self.version = version
+        self.host = "graph.facebook.com/"
+        self.url = "https://{}".format(self.host)
```

Comment:
Maybe call this `self.base_url`, and precompute the value here, instead of repeatedly computing this in the request method?